src/textual/_import_app.py

```diff
@@ -16,6 +16,23 @@ class AppFail(Exception):
     pass
 
 
+def shebang_python(candidate: Path):
+    """Does the given file look like it's run with Python?
+
+    Args:
+        candidate (Path): The candidate file to check.
+
+    Returns:
+        bool: ``True`` if it looks to #! python, ``False`` if not.
+    """
+    try:
+        with candidate.open("rb") as source:
+            first_line = source.readline()
+        return first_line.startswith(b"#!") and b"python" in first_line
```

Comment:
Could you put the return after the except? Just to keep the exception block as focused as possible.

Reply:
Aye, good call, will do.

Reply:
Done.